etc/servo_automation_screenshot.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2018 The Servo Project Developers. See the COPYRIGHT
+# file at the top-level directory of this distribution.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+"""
+Created on Mon Mar 26 20:08:25 2018
+@author: Pranshu Sinha, Abhay Soni, Aayushi Agrawal
+"""
+
+"""
+The below program is intended to test rendering mismatches in servo by taking screenshots of rendered html files.
+Here is the breakdown of how our code works:
+*   A session is started on localhost:7002
+*   The randomly generated webpage's (html files) data is sent as JSON to this session
+*   Using curl request, we load the html files for this session ID based on the session we just created.
+"""
+import os
+# import subprocess
+import json
+import requests
+import start_servo
+import time
+import base64
+import sys
+import getopt  # for command line interaction
+# import socket
+
+
+port = ''
+resolution = ''
+url_cl = ''
+file_url = ''
+
+
+def main(argv):  # take inputs from command line by considering the options parameter i.e -h, -p etc.
+    try:
+        # input options defined here.
+        opts, args = getopt.getopt(argv, "hu:p:i:r:", ["url=", "port=", "ifile=", "resolution="])
+    except getopt.GetoptError:
+        # an error is generated if the options provided in commandline are wrong.
+        # The help statement is printed on how to input command line arguments.
+        print('python3 etc/servo_automation_screenshot.py -u <url> -p <port> -i <html_file_url> -r <resolution>')
+        sys.exit(2)
+    for opt, arg in opts:
+        if opt == '-h':  # -h means help. Displays how to input command line arguments
+            print('python etc/servo_automation_screenshot.py -u <url> -p <port> -i <html_file_url> -r <resolution>')
+            sys.exit()
+        elif opt in ("-p", "--port"):  # store the value provided with the option -p in port variable.
+            global port
```

Comment:
Instead of running half of the program inside of a main() function and storing values in global variables, why don't we run the entire program inside main and use local variables?